src/data/sql-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const sqlQuiz = [
+  {
+    Question:
+      "What keyword is used when the WHERE clause fails when used with aggregate functions",
+    Answer: "HAVING",
+    Distractor1: "ADD",
+    Distractor2: "DISTINTC",
+    Distractor3: "LIMIT",
+    Explanation:
+      "The HAVING keyword was introduced because the WHERE clause fails when used with aggregate functions. So, you have to use the HAVING clause with aggregate functions instead of WHERE. With the HAVING clause, you can arrange the data in your database into many groups when you use it with the GROUP BY keyword. So, you can use it in a large database.",
```

Comment:
This is a good explanation but let's cut it down to just two sentences because long explanations will become crowded in the answer modal.